autoload/tagbar.vim

```diff
@@ -2866,6 +2866,46 @@ function! s:EscapeCtagsCmd(ctags_bin, args, ...) abort
     return ctags_cmd
 endfunction
 
+" python simulate system() on window to prevent temporary window creation
+function! s:system_python(cmd, version) abort
+    if has('nvim')
```

Comment:
Does Neovim not have this problem on Windows?

Reply:
yes, neovim doesn't need this.